eland/operations.py

```diff
@@ -285,6 +287,50 @@ def _metric_aggs(
                             results[field] = response["aggregations"][
                                 "percentiles_" + field
                             ]["values"]["50.0"]
+
+                            # If 0-length dataframe we get None here
+                            # TODO investigate if we should resolve in elasticsearch-py issue?
```

Comment:
I'm not sure if this can be solved in elasticsearch-py? The JSON result is `null` if there are no docs in the aggregation so we'll have to handle it here.